### DISCUSSION TITLE
MicroPython Firmware with SPIRAM support getting SPI FAST REBOOT in Wrover IE

### DISCUSSION THREAD

**Discussion by aurosoares** [micropython/micropython#11878] [ESP32]
Hello everyone, 

I wrote a software in micropython for an application to run on an ESP32 Wrover IE, this board supports SPIRAM. However, when using the generic ESP32 firmware the software works normally, but when using the SPIRAM firmware my ESP keeps rebooting every second. I already discarded software, just by opening PUTTY and running some simple commands I already reboot with the SPIRAM firmware(SPI FAST REBOOT). 
I'm strictly following this site: https://micropython.org/download/esp32spiram/

**Comment by davefes** [comment]
What does `esptool.py  --chip esp32 chip_id' show?
A link to the board you are using might be helpful.

**Reply by aurosoares** [reply]
The link of my board: https://www.digikey.com.br/pt/products/detail/espressif-systems/ESP32-WROVER-IE-16MB/11613178
I just buy the IC and integrate to my own board. Initially I even thought it could be a pin polarity problem in the fw upgrade, but with the generic esp32 firmware without spiram everything works perfectly.

Follow a image  upgrading with esp32 without spiram fw(with spiram the console showi the same messages):
![esptool](https://github.com/micropython/micropython/assets/23020134/a376cb65-67ed-4e6e-bb3d-780f6566636f)

**Comment by davefes** [comment]
There has been previous discussion on N16R8.  It seems to be more difficult to get working correctly then the 8M SPIRAM version that I use.
There is a lot of reading searching spiram

Good luck
